Backend/DataAccessLayer/Models/ApiKey.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace DataAccessLayer.Models
+{
+    public class ApiKey
+    {
+        public ApiKey()
+        {
+            Key = "";
+            IsUsed = false;
+        }
+
+        [Key]
+        public string Key { get; set; }
```

Comment:
The primary key should be a unique identifier such as a GUID to keep consistency with the other models in the database.

Reply:
Ok.  I will add the GUID key to both ApiKey and Application models